mets.md

```diff
@@ -120,7 +106,8 @@ The `ID` MUST be unique inside the METS file.
 
 ```
 FILEID := ID + "_" + [0-9]{4}
-ID := "OCR-D-" + WORKFLOW_STEP + ("-" + PROCESSOR)?
+ID := "OCR-D-" + WORKFLOW_STEP + ("-" + FEATURE)* + ("-" + PROCESSOR)?
+FEATURE := ("BIN" | "CROP" | "DESKEW" | DESPECK" | "DEWARP" )
 WORKFLOW_STEP := ("IMG" | "SEG" | "OCR" | "COR")
 PROCESSOR := [A-Z0-9\-]{3,}
```

Comment:
```suggestion
ID := FILEGRP + (".IMG")?
```

Reply:
> artificial "aestheticised" scheme

😆 I just love pseudo-backus-naur grammars, but I see your point.

`ID := FILEGRP + "-IMG"` makes sense.

Reply:
> > artificial "aestheticised" scheme
> 
> :laughing: I just love pseudo-backus-naur grammars, but I see your point.

I am with you: it's a shame letting pretty BNFs that look natural go to waste. But users and implementors have been avoiding that scheme. We now have the fortunate opportunity to introduce something simpler that is also more workable.

> `ID := FILEGRP + "-IMG"` makes sense.

Glad you agree. Do you want me to replace `.` with `-` in my suggestions, or are you going to reformulate anyway?